src/views/Register/__snapshots__/Register.test.jsx.snap

```diff
@@ -0,0 +1,70 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Should this file be ignore in .gitignore? since it's used for testing only

Reply:
Snapshot files are required to be included in order for snapshot tests to pass. Since we're comparing the rendered elements